scripts/build/build.ps1

```diff
@@ -12,7 +12,7 @@ $ErrorActionPreference="Stop"
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
-_ "$RepoRoot\scripts\build\generate-version.ps1"
+. "$RepoRoot\scripts\build\generate-version.ps1"
```

Comment:
it's super nice :smiley: Improves my powershell experience by a factor of 2 - that was the most annoying thing for me in powershell and this is nicely fixing it :+1: